templates/minimal/node/src/impl_fake_runtime_apis.rs

```diff
@@ -0,0 +1,98 @@
+#![allow(unused_variables)]
+
+pub struct FakeRuntime;
+use crate::types::{AccountId, Header, Nonce, OpaqueBlock as Block};
+
+sp_api::impl_runtime_apis! {
```

Comment:
We can also manually implement this stuff without this macro, but this macro already does the job easily.